File: integration-tests/TestAPI/bindings/c-sharp/Cricut.TestAPI.Tests/ExtensionTests.cs
using System;
using Xunit;

namespace Cricut.TestAPI.Tests {
    public class ExtensionTests {
        [Fact]
        void testExtensionTypes() {
            Assert.Equal(42, new String_PuttingTypesIntoQuestionablePlaces("").TestCall());
            Assert.Equal(43, Structs_PuttingTypesIntoQuestionablePlaces.Create().TestCall());
            Assert.Equal(44, new UnicodeScalar_PuttingTypesIntoQuestionablePlaces.Thing().TestCall());
        }
    }
}

File: integration-tests/TestAPI/bindings/kotlin/src/test/kotlin/com/cricut/testapi/ExtensionTests.kt
package com.cricut.testapi

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.Assertions.*
import org.junit.jupiter.api.assertThrows

class ExtensionTests {
    @Test
    fun testExtensions() {
        assertEquals(42, String_PuttingTypesIntoQuestionablePlaces("").testCall());
        assertEquals(43, Structs_PuttingTypesIntoQuestionablePlaces.create().testCall());
        assertEquals(44, UnicodeScalar_PuttingTypesIntoQuestionablePlaces.Thing.testCall());
    }
}